### ISSUE TITLE
Crash report: `Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.`

### ISSUE THREAD

**Issue by ivanwoodson** [Uniswap/interface#2420]
## URL
  
https://app.uniswap.org/#/add/0x15d4c048f83bd7e37d49ea4c83a07267ec4203da/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48/10000

## `mintV3` state
    
```json
{
  "independentField": "CURRENCY_A",
  "typedValue": "",
  "startPriceTypedValue": "",
  "leftRangeTypedValue": "0.052107",
  "rightRangeTypedValue": "0.211290"
}
```

## Error

```
Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

## Stacktrace

```
Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at df (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1793740)
    at Sa (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1765468)
    at https://app.uniswap.org/static/js/main.2b20d3db.chunk.js:1:443248
    at Lf (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1805517)
    at t.unstable_runWithPriority (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1823330)
    at qi (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1745688)
    at Rf (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1804978)
    at vf (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1796435)
    at https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1745911
    at t.unstable_runWithPriority (https://app.uniswap.org/static/js/4.98825a2c.chunk.js:2:1823330)
```

## Device data

```json
{
  "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.54 Safari/537.36",
  "browser": {
    "name": "Chrome",
    "version": "94.0.4606.54",
    "major": "94"
  },
  "engine": {
    "name": "Blink",
    "version": "94.0.4606.54"
  },
  "os": {
    "name": "Mac OS",
    "version": "10.13.6"
  },
  "device": {},
  "cpu": {}
}
```

**Comment by ivanwoodson** [comment]
All accounts are signed out. Access was tired using a different IP address and I can’t access. I feel as I am blocked from uniswap.

**Comment by moodysalem** [comment]
this piece of code is infinite looping
https://github.com/Uniswap/interface/blob/53da48b646058929b78c867469eafed90b3b8431/src/components/LiquidityChartRangeInput/Brush.tsx#L103-L107

**Comment by moodysalem** [comment]
maybe related to https://github.com/Uniswap/interface/issues/2358

**Comment by ivanwoodson** [comment]
Moody Salem I experienced the same issue when switching devices. I had the same issue using iPhone X, iPad Pro, iMac and a android device. I also moved assets from CoinBase Wallet to Metamask and had the same issues using the Dapp Browser as well.

**Comment by justindomingue** [comment]
proposed fix in #2422 @ivanwoodson you can test with the vercel preview (_deploying_ rn)
